src/Toml/Codec/Error.hs

```diff
@@ -37,16 +36,14 @@ import qualified Data.Text as Text
 @since 1.3.0.0
 -}
 data TomlDecodeError
-    = TrivialError
-    | BiMapError !TomlBiMapError
+    = BiMapError !TomlBiMapError
```

Comment:
Looks like we need to patch this constructor to make error messages more helpful... I've created a separate issue for this:

* https://github.com/kowainik/tomland/issues/313